docker/structs.go

```diff
@@ -0,0 +1,8 @@
+package docker
+
+import "github.com/docker/engine-api/types"
+
+// VolumeRequest is taken from
+// https://github.com/docker/engine-api/types/types.go
+type VolumeRequest types.VolumeCreateRequest
```

Comment:
@mapuri Makes sense. I can move them to the same file as it was before. But just a thought, wouldn't it be better to isolate all the volume related structures in a single file. I see that struct 'Volume' is defined in a different file and 'volumeGetRequest' in some other. Please let me know your thoughts @mapuri @erikh